bitsandbytes/autograd/_functions.py

```diff
@@ -455,13 +464,6 @@ def backward(ctx, grad_output):
                 CB = state.CB.to(ctx.dtype_A, copy=True).mul_(state.SCB.unsqueeze(1).mul(1.0 / 127.0))
                 grad_A = torch.matmul(grad_output, CB).view(ctx.grad_shape).to(ctx.dtype_A)
             elif state.CxB is not None:
-
-                if state.tile_indices is None:
-                    order, tile_size = state.formatB, state.get_tile_size()
-                    transform = lambda x: F.transform(x.cuda(), from_order="row", to_order=order)[0].to(x.device)
```

Comment:
There was a slight possibility of a bug before: .cuda() might transfer the tensor to GPU0 instead of the GPU we're using for weights